FlowCrypt/Models/ClientConfiguration.swift

```diff
@@ -20,6 +20,13 @@ enum ClientConfigurationFlag: String, Codable {
     case useLegacyAttesterSubmit = "USE_LEGACY_ATTESTER_SUBMIT"
     case defaultRememberPassphrase = "DEFAULT_REMEMBER_PASS_PHRASE"
     case hideArmorMeta = "HIDE_ARMOR_META"
+    case forbidStoringPassphrase = "FORBID_STORING_PASS_PHRASE"
+
+    case unknown
```

Comment:
I presume this is for forward-compatibility, so that newly added flags in the received JSON don't crash the app?

Reply:
yes, parsing of ClientConfiguration was failing, I've added tests for this case.